src/dbcp/transform/acp_projects.py

```diff
@@ -141,12 +141,19 @@ def _col_transform_owner_types(ser: pd.Series, full_df: pd.DataFrame) -> pd.Seri
     # Map "Investor Owned" values to "IPP" when the owner is likely
     # not a utility company. In the future, "Investor Owned" could refer
     # to "Utility: IOU" instead of "IPP"
-    investor_owned_owner_type = out[out == "Investor Owned"]
+    n_max_investor_owned_owner_type_per_date = (
+        full_df[full_df["raw_owner_types"].str.contains("Investor Owned")]
+        .groupby("report_date")
+        .size()
+        .max()
+    )
     n_known_investor_owned = 1
```

Comment:
Not blocking, but I would be interested in how one can determine the known number of investor owned project